app/models/network_port.rb

```diff
@@ -0,0 +1,17 @@
+class NetworkPort < ActiveRecord::Base
+  include NewWithTypeStiMixin
+  include ReportableMixin
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+  belongs_to :cloud_tenant
+  belongs_to :cloud_network
+  belongs_to :cloud_subnets
```

Comment:
Yes, good catch with the cloud_subnet

Yeah, the OpenStack infra has a tenant, we don't refresh it now, currently everything is under 'admin' tenant, but it's OpenStack, so I can't say, we will not use multiple tenants in the future. :-)